rest-api/src/main/java/gov/cms/qpp/conversion/api/services/CpcFileServiceImpl.java

```diff
@@ -29,6 +33,22 @@
 		return transformMetaDataToUnprocessedCpcFileData(metadata);
 	}
 
+	/**
+	 * Retrieves the file location id and retrieves the file if it is an unprocessed cpc+ file
+	 *
+	 * @param fileId {@link Metadata} identifier
+	 * @return file returned as an {@link InputStream}
+	 */
+	public InputStream getFileById(String fileId) {
+		InputStream inputStream = new ByteArrayInputStream("File not found!".getBytes());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getBytes" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1943)